src/index.html

```diff
@@ -11,7 +11,207 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    id="form"
+    name="form"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset 
+      class="personal__information" 
+      name="personal__information">
+        <legend>Personal information</legend>
+      
+        <div class="information">
+          <label for="username">Surname:</label>
+          <input 
+          id="username" 
+          type="text"
+          name="username" 
+          required
+          >
+        </div>
+        <div class="information">
+          <label for="name">
+            Name:
+          </label>
+          <input 
+          id="name" 
+          type="text"
+          name="name" 
+          required
+          >
```

Comment:
Follow the requirements, please 
<img width="484" alt="image" src="https://user-images.githubusercontent.com/99403198/216770115-69d3fc64-5f0e-4238-867f-9287337ae9c1.png">